tez-runtime-internals/src/main/java/org/apache/tez/runtime/metrics/TaskCounterUpdater.java

```diff
@@ -48,11 +49,16 @@ public class TaskCounterUpdater {
   private final TezCounters tezCounters;
   private final Configuration conf;
 
+//  /**
+//   * A Map where Key-> URIScheme and value->FileSystemStatisticUpdater
+//   */
+//  private Map<String, FileSystemStatisticUpdater> statisticUpdaters =
+//     new HashMap<>();
```

Comment:
why are you commenting this out? Can't we just delete it

Reply:
oh right, leftover, removing it